pytorch_blade/src/ltc/disc_compiler/passes/disc_fuser.cpp

```diff
@@ -61,9 +61,8 @@ void CastBoundaryScalarToTensor(
   auto orig_input = disc_graph->inputs()[i + 1];
   auto item_node = disc_graph->create(aten::item, {new_input});
   item_node->output()->setType(getScalarTypePtr(typ));
-  disc_graph->appendNode(item_node);
+  disc_graph->prependNode(item_node);
```

Comment:
Is there any problem with the previous implementation?

Reply:
Can we extend this function with one more argument that specifies how to add node(prepend or append), so we cat reuse it in CastingScalarInputToTensor & CastingScalarOutputToTensor